src/trackers/AVISTAZ_NETWORK.py

```
@@ -42,7 +42,10 @@ def __init__(self, config, tracker_name):
         }, timeout=60.0)
         self.media_code = ''
 
-    def get_resolution(self, meta):
+    async def rules(self, meta: dict[str, Any]) -> str:
```

Ruff (0.14.10)

Unused method argument: `meta`

(ARG002)